README.md

```diff
@@ -373,15 +373,22 @@ will give a more in-depth description of the command and available options.
 #### Proxy Submissions
 
 The autograder allows course staff to submit on behalf of students using "proxy" submissions.
-The proxy submit command adds two parameters to the normal submission command:
+The proxy submit commands add two parameters to the normal submission command:
 
  - `proxy-email` -- The student that the course staff is submitting on behalf of.
  - `proxy-time` -- The "fudged" submission time that is automatically set to the earlier time between now and one minute before the due date.
    - If you want to set the time manually, time is the number of milliseconds from the [Unix Epoch](https://en.wikipedia.org/wiki/Unix_time).
 
+In future examples with `proxy-time`, we can use a [Unix Epoch time converter](https://www.epochconverter.com/) to change the target time
+of March, 4th 2025 at 12:00 PM Pacific Time to milliseconds from the Unix Epoch, `1741118400000`.
+
 As the commands come from course staff, the submissions are not subject to submission restrictions.
-Here is an example usage of `autograder.cli.courses.assignments.submissions.proxy.submit` where
-course staff submits code (`my_file.py`) on behalf of `student@test.edulinq.org`.
+
+##### Submitting Code Files for a Student
+
+Proxy submit files (code) for a student by using the `autograder.cli.courses.assignments.submissions.proxy.submit` command.
```

Comment:
Consider: s/files (code)/source files/

(If you choose this, then you may want to change the subsection title as well.)